..env.development.local

```diff
@@ -0,0 +1,2 @@
+REACT_APP_WEBMAP_DOMAIN=http://dev.treetracker.org
```

Comment:
I don't think you meant to commit this 😉

Reply:
Oops! Good catch, thanks!